static/app/utils/getKeyCode.ts

```diff
@@ -0,0 +1,79 @@
+/* key maps and utils retrieved from  https://github.com/jaywcjlove/hotkeys */
+
+/** Includes a lot of leftover unused codes for the future in case we want glyphs for them */
+const _keyMap = {
```

Comment:
`_` has a special meaning typically (turns off the unused lint check for variables).

This can probably just be `keyNameCodeMapping` or something like that?